apps/saphana.go

```diff
@@ -16,9 +16,117 @@ package apps
 
 import (
 	"github.com/GoogleCloudPlatform/ops-agent/confgenerator"
+	"github.com/GoogleCloudPlatform/ops-agent/confgenerator/fluentbit"
 	"github.com/GoogleCloudPlatform/ops-agent/confgenerator/otel"
 )
 
+type LoggingProcessorSapHanaTrace struct {
+	confgenerator.ConfigComponent `yaml:",inline"`
+}
+
+func (LoggingProcessorSapHanaTrace) Type() string {
+	return "saphana_trace"
```

Comment:
Are there other logs that are not `trace` that make its way to `syslog`? Or that could be configured in the future?
I'm wondering why we don't just call this `saphana` like we do with other receivers that have only a singular input

Reply:
Sure, we can change it.